react-frontend/src/css/nav.css

```diff
@@ -17,7 +17,23 @@ header {
     margin: auto;
     width: 1115px;
   }
-  
+
+  .covidBanner {
```

Comment:
This is part of the pattern we are hoping to move away from.  New css should be implemented using styled components.  The goal is to prevent huge css files.